daml-lf/language/daml-lf.bzl

```diff
@@ -23,7 +23,7 @@ LF_VERSIONS = [
 
 # The subset of LF versions accepted by the compiler in the syntax
 # expected by the --target option.
-COMPILER_LF_VERSIONS = ["1.dev" if ver == "dev" else ver for ver in LF_VERSIONS if ver != "1.6"]
```

Comment:
I thought we stopped supporting 1.6 in the compiler but apparently we didn’t. We don’t support it in daml script and triggers since those rely on `Any`.

Reply:
How do you feel about introducing a `SCRIPT_LF_VERSIONS` and/or `TRIGGER_LF_VERSIONS` below this to filter out "1.6", instead of filtering 1.6 everywhere else?

Reply:
Great idea, I added the filters one after the other so I didn’t fully realize how often I repeated myself. I’ve only introduced SCRIPT_LF_VERSIONS since they use the same and the differentiation didn’t seem super helpful. Triggers depend on script so they can never be more general.

Reply:
That makes a lot of sense!